model/BUILD.bazel

```diff
@@ -66,7 +66,8 @@ proto_library(
 
 java_lite_proto_library(
     name = "onboarding_java_proto_lite",
-    deps = [":onboarding_proto"],
+    visibility = ["//visibility:public"],
+    deps = [":onboarding_proto"],   
```

Comment:
there is whitespace at the end of this failing with causing the failure

<img width="435" alt="Screenshot 2021-02-14 at 09 52 46" src="https://user-images.githubusercontent.com/25057618/107868441-69980c00-6eaa-11eb-9f8a-ec082c312a2f.png">